addons/core/tests/integration/helpers/relative-datetime-live-test.js

```diff
@@ -0,0 +1,38 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+
+module('Integration | Helper | relative-datetime-live', function (hooks) {
+  setupRenderingTest(hooks);
+
+  test('it renders relative date in "time" ago format', async function (assert) {
```

Comment:
Maybe show a few examples of a date in the future as well?

Reply:
@ZedLi meaning?

Reply:
As in some tests with the date in the future to show it works not just with dates in the past

Reply:
@ZedLi fixed